test/integration/crud/crud.prose.test.ts

```diff
@@ -188,12 +189,12 @@ describe('CRUD Prose Spec Tests', () => {
       client.on('commandStarted', filterForCommands('bulkWrite', commands));
       commands.length = 0;
 
-      Array.from({ length: maxWriteBatchSize + 1 }, () => {
-        models.push({
+      models = Array.from({ length: maxWriteBatchSize + 1 }, () => {
```

Comment:
These `Array.from` changes are unrelated to the TS changes, just making them from a suggestion on the previous PR for the other test.